src/rental.rb

```diff
@@ -1,6 +1,5 @@
 class Rental
-  attr_accessor :date
-  attr_reader :book, :person
+  attr_accessor :date, :book, :person
```

Comment:
- [ ] I found this error after running your project so I think it would be great if you check out and try to solve the problem there 🚀💯💯 so to let the user be able to run your program 🚀 

![image](https://user-images.githubusercontent.com/58553711/202075876-a119c5b0-3a6d-4f35-a7f5-158caf3cae42.png)